src/utils/hints/contractVerifiedHints.js

```diff
@@ -0,0 +1,7 @@
+export const contractVerifiedHints = {
+  license: 'License information for the contract.',
+  compiler: 'Compiler version used for the contract.',
+  entryFile: 'Main file for the contract source code that includes dependencies.',
```

Comment:
```suggestion
  entryFile: 'Main file of the smart contract source code',
```
I would change it like this as potentially it could not include any dependency.